src/pages/NewQuestion/components/OptionSelect/OptionSelect.styles.ts

```diff
@@ -0,0 +1,48 @@
+import styled from '@emotion/styled';
+
+const OptionSelectWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  height: 100svh;
```

Comment:
```suggestion
  height: 100vh;
  height: 100svh;
```

여기도 fallback 스타일링이 필요해요!